### ISSUE TITLE
Dependency Issue: iterativemapreduce snapshots not deployed.

### ISSUE THREAD

**Issue by drewfarris** [jpatanooga/Metronome#1]
tl; dr;
I ended up cloning git@github.com:emsixteeen/IterativeReduce.git and building locally.

I was unable to find a repo that contains 'com.cloudera:iterativereduce:jar:0.1-SNAPSHOT'

Checked out rev a83a762fce6 and ran `maven clean install`, build died with:

```
[ERROR] Failed to execute goal on project Metronome: Could not resolve dependencies 
for project tv.floe.metronome:Metronome:jar:1.0-SNAPSHOT: Failure to find 
com.cloudera:iterativereduce:jar:0.1-SNAPSHOT in https://repository.cloudera.com
/artifactory/cloudera-repos/ was cached in the local repository, resolution will not be
reattempted until the update interval of org.apache.hadoop has elapsed or updates are 
forced -> [Help 1]
```

**Comment by jpatanooga** [comment]
yup. my bad, ran low on time and didn't upload the new wiki docs.

Just got off a plane, give me a day or so and we're gonna create tags for the project for a 0.1v and then a downloadable version thats already built. Of course if you are on a different hadoop distro / cdh version, then you really need to re-compile (which I'll cover in the docs very soon).

beyond that just email me directly for questions and I'll get you going. apologies.

**Comment by drewfarris** [comment]
No worries, I wanted to make sure it was documented in case other folks run
into the same problem.

Thanks for the quick response and interesting talk.

**Comment by jpatanooga** [comment]
Cool. yeah, the next two weeks are dedicated solely to packaging and more
documentation on the examples. Adam is a bit ahead of me on docs on the
core DL4J site.
